### PR TITLE
Feature/ TK 36 Develop Front-end Integration to Challenge Details When Entering Online Challenges

### PR DESCRIPTION
**Pull request by illuminatyy**
# What was done? ✅
> Describe in a few words what have you done? 
> - Step 1: Online Challenge Details Integrated when tapping an online challenge
> - Step 2: Participants List is calling the originals members of challenge
> - Step 3: Invite friends Integrated by tapping the Online Challenge Details button
> - Step 4: Adjusted localizable files
# Description 📝
  
# Evidence 🕵️‍♀️
## **Screenshots/Videos/Figma** 📱
https://user-images.githubusercontent.com/51174686/198375171-8694434c-3e66-495d-bb43-29c3fcfec48f.mp4

### CODE REVIEW

File: Fiero/Views/Challenges/EnterWithCodeView.swift [hunk 1]
```diff
@@ -9,7 +9,7 @@ import SwiftUI
 import Combine
 
 struct EnterWithCodeView: View {
-    @Environment(\.dismiss) var dismiss
+    @Environment(\.presentationMode) var presentationMode
```
Comment (MarinaFX) [inline Fiero/Views/Challenges/EnterWithCodeView.swift:12]: reverter mudanças

File: Fiero/Views/Challenges/ChallengesListScreenView.swift [hunk 2]
```diff
@@ -196,8 +199,11 @@ struct ChallengesListScreenView: View {
             VStack{
                 Spacer()
                 ButtonComponent(style: .primary(isEnabled: true), text: "Entrar por código") {
-                    self.isShowingEnterWithCodeView = true
```
Comment (MarinaFX) [inline Fiero/Views/Challenges/ChallengesListScreenView.swift:199]: no fiero amamos self, viva self supremacy

**Review by MarinaFX — CHANGES_REQUESTED** [review]

File: Fiero/Views/Challenges/ChallengesListScreenView.swift (hunk 2 above)
Comment (illuminatyy) [reply Fiero/Views/Challenges/ChallengesListScreenView.swift:199]: feito

File: Fiero/Views/Challenges/EnterWithCodeView.swift (hunk 1 above)
Comment (illuminatyy) [reply Fiero/Views/Challenges/EnterWithCodeView.swift]: feito também, em todo o código

**Review by MarinaFX — APPROVED** [review]

**Pull request merged**